### ISSUE TITLE
[FEATURE REQUEST] Improved tooltips/explanations of fields in GUI

### ISSUE THREAD

**Issue by josephrocca** [huggingface/autotrain-advanced#826]
### Feature Request

I'm pretty bad at writing docs myself, so I can't criticize, but it would seemingly take quite a small amount of effort to add hover tooltips to the fields like "Train" and "Save total limit".

You can use the HTML built in `title` attribute, so there's no need to import any libraries or anything like that. I.e. 

```
<input title="Explanation of what it does" ...>
```

### Motivation

Would make it much easier to use for newbies.

### Additional Context

Also, every time you make a mistake in the GUI, the whole space/container crashes, and it's not clear to the user what caused it - especially via huggingface spaces, because it says "the owner has paused this". And you need to know to click on the little logs button at the top to work out what happened.

A few usability improvements would make this project a lot more useful to newbies I think.

**Comment by josephrocca** [comment]
I don't want to spam issues in your repo, so I'm just going to add some more feedback here:

* Shouldn't "auto choose batch size" be a *non*-advanced feature? I.e. choosing the batch size manually seems more like an 'advanced' feature for an auto ML application. My understanding of the project is that you try to simplify things so there are fewer complicated knobs to turn to get a good result.
* It took my a long time to work out that I had to write "train" in the "Train" field, and rename my csv to end with `_train`. Each time I got it wrong, the space would crash and I'd have to wait several minutes for it to boot back up, then I'd have to re-input all the settings, and try again.

**Comment by abhishekkrthakur** [comment]
thanks for your suggestions, ill consider them and add more info to docs. 

> Also, every time you make a mistake in the GUI, the whole space/container crashes, and it's not clear to the user what caused it - especially via huggingface spaces, because it says "the owner has paused this". And you need to know to click on the little logs button at the top to work out what happened.

To avoid this, when creating the space, set PAUSE_ON_FAILURE to 0

**Comment by josephrocca** [comment]
> To avoid this, when creating the space, set PAUSE_ON_FAILURE to 0

I'd you're able to make that default it would be a big UX improvement

**Comment by abhishekkrthakur** [comment]
not really. the choice is presented when creating the space. its an advanced option for debugging. people dont want to get charged for having left the space running with nothing happening. also, the logs are available even when space goes to sleep mode on error.

**Comment by josephrocca** [comment]
> also, the logs are available even when space goes to sleep mode on error.

Yes, but the UX here is quite bad. It just says "The owner paused this space" or something. And you need to know to click the little icon in the top bar. It's not clear at all why it happened, and doesn't say to check the logs to find the cause.

If you want a default here for preventing unwanted spending, it'd be better to have/use the "pause if no GPU activity after N hours".

I understand that this is platform-level stuff though, and you might not have direct control over this stuff.
